library/src/main/java/com/github/pwittchen/reactivebus/library/Event.java

```diff
@@ -73,27 +93,27 @@ public int hashCode() {
     return Objects.hash(id, name);
   }
 
-  public static class Builder {
+  public final static class Builder {
     private String id = UUID.randomUUID().toString();
     private String name = "";
     private Serializable data;
 
```

Comment:
Prevent instantiation of Builder by adding private do-nothing constructor:
```java
private Builder() {
}
```

Reply:
Fixed.